hnn_core/tests/test_network.py

```diff
@@ -83,6 +83,48 @@ def test_network_models():
         assert np.all(np.diff(k_gbar, n=2) > 0)  # positive 2nd derivative
 
 
+def test_network_cell_positions():
+    """"Test manipulation of cell positions in the network object"""
+
+    net = jones_2009_model()
+    assert np.isclose(net._inplane_distance, 1.)
+    assert np.isclose(net._layer_separation, 1307.4)
+    pos_dict_nodrives = deepcopy(net.pos_dict)
+
+    net.set_cell_positions(inplane_distance=2.)
+    # check that in-plane distance has doubled from the default 1.0
+    assert np.isclose(2. * pos_dict_nodrives['L5_pyramidal'][1][1],
+                      net.pos_dict['L5_pyramidal'][1][1])
```

Comment:
sorry to ask again. Let me try to rephrase. Isn't `pos_dict` the position not the in-plane distance? For checking the inplane distance you will have to actually do `np.diff` or some such thing

Reply:
I see the confusion now, the test is formulated weirdly. I've made a better one using `diff`